.github/workflows/publish_arm.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish ARM
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - dockerfile: ./arm/Dockerfile
+            image: ghcr.io/${{ github.repository }}
```

Comment:
I believe this one is used as the name of the docker image so it should be different than the one we are using for ubuntu no?

Reply:
The strategy here is to use the same image path/name, but differentiate the images in how they're tagged. See the [results](https://github.com/threefoldtech/rmb-rs/pkgs/container/rmb-peer) for the rmb-peer images for reference. It's actually possible to have multiple architectures under one tag, but that requires building all images in one workflow which I wanted to avoid at this point.

Reply:
When pulling the image does it automatically pull the right one?

Reply:
For the regular (amd64) images, nothing changes. These images are already tagged with the version number, for example, ghcr.io/threefoldtech/farmerbot:**0.2.0**

Using the ARM images requires adding the suffix `-arm`, for example, ghcr.io/threefoldtech/farmerbot:**0.2.0-arm**

So this requires having a separate `docker-compose.yaml` containing the ARM tagged images. But generally speaking, yes, Docker can automatically choose the correct image for a given platform when multiple architectures are published under one tag.